### PR TITLE
3.3.10

### LINKED ISSUE #125: jdom issues for 3.3.10
Hi @yihangx , I reported these issues first in Slack but @AlexanderPico mentioned to me you're not in that channel. I recently checked out the code in this repo to work on #123 myself, however I could not compile the app from the command line (mvn install resulted in errors). After some digging, I found that several maven artifacts required a higher version in the pom.xml, then which were added in the localSettings.sh file (leading to this PR #124 ). But, the jdom artifact is still causing me issues.

1. The pom.xml requires version 2.0.1 for com.springsource.org.jdom-2.0.1.jar
2. Several other users have reported issues with the jdom artifact version 2.0.1 not being downloadable: https://groups.google.com/a/opencast.org/g/dev/c/J7SU8zsM5SA ; I recently cleaned out my .m2 folder, so don't have an old version of this artifact lying about unfortunately
3. It seem that this specific jdom maven library is not maintained anymore, so that might need to be replaced (in the long run) for the WikiPathways-Cytoscape app.
4. According to the jdom website, there is a 2.0.6.1 version: http://www.jdom.org/ , And the jdom github repo seems to be maintained: https://github.com/hunterhacker/jdom/ . The artifact is now called jdom2, but should still be an XML parser.
5. I'm trying to get this maven artifact (https://mvnrepository.com/artifact/org.jdom/jdom2/2.0.6.1) in my localSettings, however I have been unsuccessful for the time being: 
```
mvn install:install-file -Dfile=localRepo/org.jdom-2.0.6.1.jar -DgroupId=org.jdom -DartifactId=jdom2 -Dversion=2.0.6.1 -Dpackaging=jar
```
The error might be cause by the 'path-to-file' not being correct, or another issue which I cannot see right now, so any help would be much appreciated @yihangx (and also @mkutmon :smile: )

### PR DESCRIPTION
**Pull request by DeniseSl22**
@AlexanderPico @mkutmon @yihangx : I believe I resolved the jdom issue :D 
I have a stable build now, which works in Cytoscape.
Resolves #125

### CODE REVIEW

File: localsetup.sh [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # pathvisio core
-mvn install:install-file -Dfile=localRepo/org.pathvisio.core.jar -DgroupId=org.pathvisio -DartifactId=pathvisio-core -Dversion=3.2.2 -Dpackaging=jar
+mvn install:install-file -Dfile=localRepo/org.pathvisio.core.jar -DgroupId=org.pathvisio -DartifactId=pathvisio-core -Dversion=3.3.0 -Dpackaging=jar
 
 # bridgedb 
-mvn install:install-file -Dfile=localRepo/org.bridgedb.jar -DgroupId=org.bridgedb -DartifactId=bridgedb -Dversion=2.0.0 -Dpackaging=jar
-mvn install:install-file -Dfile=localRepo/org.bridgedb.bio.jar -DgroupId=org.bridgedb -DartifactId=bridgedb-bio -Dversion=2.0.0 -Dpackaging=jar
+mvn install:install-file -Dfile=localRepo/org.bridgedb.jar -DgroupId=org.bridgedb -DartifactId=bridgedb -Dversion=3.0.13 -Dpackaging=jar
```
Comment (egonw) [inline localsetup.sh:5]: I would suggest to remove this and the other `mvn install:install-file` lines for BridgeDb and let Maven figure it out with the regular Maven resolution.

Comment (DeniseSl22) [reply localsetup.sh:5]: Yes, great suggestion for future version! But for now, I think this is the most stable way to get the app building

Comment (DeniseSl22) [reply localsetup.sh:5]: @egonw : see also the documentation:

https://github.com/wikipathways/cytoscape-wikipathways-app/wiki/Add-local-dependency

**Comment by yihangx** [comment]
@DeniseSl22 Thanks for resolving this! Merged.

**Pull request merged**